src/DotNetWorker.Core/Hosting/AutoGeneratedClassAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    /// <summary>
+    /// Identifies class generated by Azure functions worker SDK.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class)]
+    public sealed class AutoGeneratedClassAttribute : Attribute
```

Comment:
I originally named this as `FunctionsWorkerSdkGeneratedClassAttribute`, but later felt that it is somewhat redundant with the namespace. So switched to `Auto` prefix. (Same for the method attribute as well). Let me know you think we should use a different prefix/name.